README.md

```diff
@@ -1,25 +1,39 @@
 Riak .NET Client - a .NET client for Riak
 =========================================
 
-**Riak .NET Client** is a .NET library which makes it easy to communicate with [Riak](http://riak.basho.com/), [Basho](http://www.basho.com/)'s fault-tolerante, distributed KV store.
-
+**Riak .NET Client** is a .NET library which makes it easy to communicate with [Riak](http://basho.com/riak/), an open source, distributed database that focuses on high availability, horizontal scalability, and *predictable*
+latency. Both Riak and this code is maintained by [Basho](http://www.basho.com/).
+
+1. [Installation](#installation)
```

Comment:
Wiki has been updated - https://github.com/basho/riak-dotnet-client/wiki/Installation - this section is removed from the PR.